client/onboarding/tasks/add-business-info-task/required-verification-info.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Card, CardBody } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { LoadableBlock } from 'components/loadable';
+import { useRequiredVerificationInfo } from 'data/onboarding';
+import Requirements from 'onboarding/requirements';
+
+const RequiredVerificationInfo = ( { country, type, structure } ) => {
+	const {
+		requiredFields,
+		isLoading,
+		getRequiredVerificationInfo,
+	} = useRequiredVerificationInfo( {
+		country,
+		type,
+		structure,
+	} );
+
+	useEffect( () => {
+		getRequiredVerificationInfo( { country, type, structure } );
+	}, [ country, type, structure, getRequiredVerificationInfo ] );
+
+	return (
+		<Card size="large" className="wcpay-required-info-card">
+			<CardBody>
+				<p>
+					<b>
+						{ __(
+							"To verify your details, we'll require:",
```

Comment:
We talked about using `Verifying your company can require:` instead here, thinking that those values can be outdated.

Reply:
Great catch! I updated in c4992ad835df9df811d7dd8639be73fba2dea6ce, but went with the slightly more generic `Verifying your details can require:`. I also updated the main description to say `we may` instead of `we will` when talking about the requirements.

Let me know what you think! If you think the original string works better here I'm happy to update it back.